### PR TITLE
ISSUE-217 qudt alignment

### PR DESCRIPTION
**Pull request by dr-shorthair**
Ran Agreement-Maker but it didn't find a lot of matches, while getting a lot wrong. 
So I reverted to a manual process. 

This graph contains alignments from [reprSciUnits](https://github.com/ESIPFed/sweet/blob/master/src/reprSciUnits.ttl) to [QUDT Units](https://github.com/qudt/qudt-public-repo/blob/master/vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl) and [QUDT Prefixes](https://github.com/qudt/qudt-public-repo/blob/master/vocab/prefixes/VOCAB_QUDT-PREFIXES-v2.1.ttl). 
Most can be matched, and I've advised QUDT of the ones that are missing from their vocabulary - see https://github.com/qudt/qudt-public-repo/issues/231 

All except one alignment uses `skos:exactMatch` as the alignment predicate. 
For `reprSciUnits:year` the conversion factor is different to all four of the definitions for Year in QUDT so I just left it at `skos:closeMatch` for now. 

There will also be potential for alignment to [QUDT QuantityKinds](https://github.com/qudt/qudt-public-repo/blob/master/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl) but that looks like quite a big job. 

Related to #217

### CODE REVIEW

**Comment by carueda** [comment]
@dr-shorthair I just noted this review request ... sorry, I won't be able to review for the time being.  Since I'm noting 3 reviewers now and that 3 reviews are required for merge, may I ask you to consider other possible extra reviewers?

File: alignments/sweet-qudt-mapping.ttl [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+  skos:exactMatch <http://qudt.org/vocab/unit/WB> ;
+.
+reprSciUnits:week
+  skos:exactMatch <http://qudt.org/vocab/unit/WK> ;
+.
+reprSciUnits:year
+  skos:closeMatch <http://qudt.org/vocab/unit/YR> ;
+  skos:closeMatch <http://qudt.org/vocab/unit/YR_Common> ;
+  skos:closeMatch <http://qudt.org/vocab/unit/YR_Sidereal> ;
+  skos:closeMatch <http://qudt.org/vocab/unit/YR_TROPICAL> ;
```
Comment (graybeal) [inline alignments/sweet-qudt-mapping.ttl:389]: Well, it's definitely picking nits, but I would suggest making YR_TROPICAL a closeMatch, and the other three a relatedMatch. Reason being that YR_TROPICAL is quite a bit closer numerically than the other 3, thus more likely to have been the intended meaning of the SWEET term for 'year'. The others are clearly different numbers.  (If only there had been a definition in SWEET...)

**Review by graybeal — CHANGES_REQUESTED** [review]
Is good work. You don't have to take my input on the changes to the Year mapping, but I think it's a bit of an improvement that saves other people the trouble of looking it up the way I just did.

File: alignments/sweet-qudt-mapping.ttl (hunk 1 above)
Comment (dr-shorthair) [reply alignments/sweet-qudt-mapping.ttl]: Thanks John - like you say, if only we knew what SWEET intended

**Review by lewismc — APPROVED** [review]
+1 from me. I went through more than half of the alignments and they all seemed sensible to me.
I have a non-blocking side question for you @dr-shorthair, was there a version of QUDT that you aligned SWEET master branch against? If so, do you have any idea of how quickly this alignment may become stale... if at all? 
Maybe the next step would be for us to work on automating an alignment if at all possible... this is easy to ponder and difficult to implement but I thought I'd ask it none-the-less. Thank you

File: alignments/sweet-qudt-mapping.ttl (hunk 1 above)
Comment (dr-shorthair) [reply alignments/sweet-qudt-mapping.ttl]: If you can resolve and approve we can move forward now I think.

**Comment by dr-shorthair** [comment]
> was there a version of QUDT that you aligned SWEET master branch against? If so, do you have any idea of how quickly this alignment may become stale... if at all?

Since v2 QUDT is maintained continuously. So I aligned SWEET to current state. I am on the QUDT TAB and am pushing to clarify the policy around revisions, but one thing that has to be non-negotiable is that any published URI will remain de-referenceable, though the representation might be just a pointer to a preferred representation. That should ensure no 404 links, i.e. the alignment should not go stale. 

As for automating the alignment - well the SWEET representation is not very expressive, so most of the work would be on SWEET side. I did this one manually - a bit tedious but I judged it would be quicker than trying to automate this time.

File: alignments/sweet-qudt-mapping.ttl (hunk 1 above)
Comment (dr-shorthair) [reply alignments/sweet-qudt-mapping.ttl]: (I accepted your suggestion and updated the code.)

**Comment by lewismc** [comment]
Understood @dr-shorthair thanks for the commentary.

**Review by nicholascar — APPROVED** [review]
manually checked alignments - seem fine!

**Comment by lewismc** [comment]
@graybeal can you please take a look at @dr-shorthair 's most recent additions? Thank you

**Comment by graybeal** [comment]
> @graybeal can you please take a look at @dr-shorthair 's most recent additions? Thank you

can't right now, no. (can't even find them, not enough time or github-fu. But I thought I made that pull request and he's included it, so there isn't anything else to review?

**Comment by dr-shorthair** [comment]
@graybeal the mapping graph is here: https://github.com/ESIPFed/sweet/blob/issue217-qudt-alignment/alignments/sweet-qudt-mapping.ttl 
I adjusted the mapping for `year` following your suggestion - https://github.com/ESIPFed/sweet/blob/issue217-qudt-alignment/alignments/sweet-qudt-mapping.ttl#L385 
Need you to switch your review to 'accept' then it can be merged.

**Review by graybeal — APPROVED** [review]

**Pull request merged**